Gemfile

```diff
@@ -1,9 +1,10 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 group :development do
-  gem "test-kitchen", :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
-  gem 'kitchen-vagrant', git: 'https://github.com/afiune/kitchen-vagrant.git', :branch => 'Transport'
-  gem "berkshelf"
-  gem "vagrant-wrapper", ">= 2.0"
-  gem "chefspec"
+  gem 'test-kitchen', :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
+  gem 'kitchen-vagrant', git: 'https://github.com/jdmundrawala/kitchen-vagrant.git', :branch => 'Transport'
```

Comment:
I don't think you should be pointing at my branch... it should probably be Salim's